transformers/uws/server.js

```diff
@@ -69,6 +69,8 @@ module.exports = function server() {
     spark.emit('incoming::data', msg);
   });
 
+  native.server.group.onPing(group, (msg, spark) => spark.heartbeat());
```

Comment:
Exactly! To be honest I'm not sure if the issue is still reproducible of if it happens with uws.